olive/passes/openvino/quantization.py

```diff
@@ -97,3 +103,38 @@ def _run_for_config(self, model: OpenVINOModel, config: Dict[str, Any], output_m
         openvino_model = OpenVINOModel(model_path)
 
         return openvino_model
+
+    def _create_dataloader(self, common_dataloader):
+        """
+        Create an openvino.tools.pot.api.DataLoader instance from a common dataloader.
```

Comment:
>dataloader [](http://example.com/codeflow?start=75&length=10)

torch Dataloader?

Reply:
It is a torch dataloader in most of our use cases and examples. But I don't think it will be the same since the `dataloader` component in `olive.data` is not specified to return a torch dataloader. 
For example, the dummy datacontainer dataloader is not torch dataloader. 
@trajepl please correct me if I am wrong.

Reply:
It is right. The `dataloader` in `olive.data`

But somehow the `Dataloader` in OpenVINO is more like the `Dataset` in torch.